test/v2/services/CreateForeignPropertyPeriodSummaryServiceSpec.scala

```diff
@@ -95,7 +95,7 @@ class CreateForeignPropertyPeriodSummaryServiceSpec extends UnitSpec {
     )),
     Some(Seq(CreateForeignNonFhlPropertyEntry("FRA",
       Some(ForeignNonFhlPropertyIncome(
-       Some( ForeignNonFhlPropertyRentIncome(Some(5000.99))),
```

Comment:
Again don't need both consolidated and regular examples - a single (very simple) one will do - as the request body is not touched by the service other than to pass it to the connector.